app/templates/gulpfile.js

```diff
@@ -231,6 +231,91 @@ gulp.task('init-local', function() {
 });
 
 /**
+ * Creates a sample process.json
+ */
+gulp.task('init-process', function() {
+  //copy from slushfile - with modifications to use config instead
+  log('Creating process.json sample document with values drawn from gulp.config.js');
+  try {
+    var configJSON = pm2JSON();
+
+    var configString = JSON.stringify(configJSON, null, 2) + '\n';
+    fs.writeFileSync('process.json', configString, {
+      encoding: 'utf8'
+    });
+  } catch (e) {
+    console.log('failed to write process.json: ' + e.message);
+  }
+});
+
+/**
+ * Creates a sample ecosystem.json
+ */
+gulp.task('init-ecosystem', function() {
+  log('Creating ecosystem.json sample document with values drawn from gulp.config.js');
+  try {
+    var configJSON = pm2JSON();
+
+    configJSON.deploy = {
+      'sample-target': {
+        'key': '/path/to/key',
+        'user': 'USERNAME',
+        'host': ['HOSTNAME'],
+        'ref': 'origin/master',
+        'repo': 'https://GIT_USERNAME:GIT_PASSWORD@marklogic.unfuddle.com/git/REPO_NAME/',
```

Comment:
Adding fingerprints, needs to be done by every uses that wants to deploy, on each of the machines they want to deploy.

Github: ssh into the server, and run `ssh -T git@github.com`, accept fingerprint
Unfuddle: no elegant way, ssh into the server, and run git clone against any git repo on unfuddle, accept fingerprint, terminate the git clone once past that stage, remove any subdir it created..